woocommerce/woocommerce-android
# Analytics performance: Introduce analytics data store for Timestamps

Summary
==========
Partially fix issue #9267 by introducing a Data store implementation allowing the app to persist all timestamps for the known selection types inside the Analytics Hub.

Pending improvements for upcoming PRs:
- Connect the Data store with the `UpdateAnalyticsHubStats` use case
- Support timestamp tracking of Custom range selections 
- Support for the My Store section

How to Test
==========
1. Review the introduced code and verify that all unit tests are green. The data store is not connected to anything yet.

Update release notes:
